linux_os/guide/services/ssh/ssh_server/sshd_enable_pubkey_auth/rule.yml

```diff
@@ -0,0 +1,41 @@
+documentation_complete: true
+
+title: 'Enable PubkeyAuthentication'
+
+description: |-
+    Verify the sshd daemon allows public key authentication with the
+    following:
+    <pre>$ grep ^Pubkeyauthentication /etc/ssh/sshd_config
```

Comment:
```suggestion
    <pre>$ grep ^PubkeyAuthentication /etc/ssh/sshd_config
```

Reply:
thanks for the suggestions, I've added it and rebased.